tools/shared/src/main/scala/org/scalajs/core/tools/corelib/CoreJSLibs.scala

```diff
@@ -132,7 +132,16 @@ object CoreJSLibs {
           .replaceAll("ScalaJS\\.isArrayOf\\.", "\\$isArrayOf_")
           .replaceAll("ScalaJS\\.asArrayOf\\.", "\\$asArrayOf_")
           .replaceAll("ScalaJS\\.", "\\$")
-          .replaceAll("\n(\\$[A-Za-z0-9_]+) =", "\nvar $1 =")
+          .replaceAll("\n(\\$[A-Za-z0-9_]+) =", "\nconst $1 =")
+    }
+
+    outputMode match {
+      case OutputMode.ECMAScript51Global | OutputMode.ECMAScript51Isolated =>
+        content1
+          .replaceAll("(let|const) ", "var ")
```

Comment:
Do you think we could check for a word boundary on the left? This seems to be very close to the clbuttic mistake...